### ISSUE TITLE
Change how the quarter period is defined

### ISSUE THREAD

**Issue by rotkiv1205** [liamcain/obsidian-periodic-notes#212]
Hello!

Love the plugin and thought I'd shoot a request I've had. 

I would benefit from being able to change how the quarter period is defined. 

My use case is that I use periodic notes as a diary-of-sorts in connection to my studies at uni. There we have 4 study periods per year which don't align with normal quarters. So like, today is the start of study period 4, for example. 

Is it currently possible to change this or is it a feature that could be added?

**Comment by arnoesterhuizen** [comment]
I do something similar to align calendar years with financial years for our company, e.g. 2024-Q1 is FY2024-Q4, and 2024-Q2 is FY2025-Q1. The financial year is the calendar year plus 9 months, so I add a simple calculation in my quarter template

```md
- {{quarter:YYYY-[Q]Q}}
- FY{{quarter+9M:YYYY-[Q]Q}}
```

**Comment by rotkiv1205** [comment]
That's nice and something along the lines of what I'm searching for! If I understood you correctl, you use those calculations in the body of a template note? Is it possible to add something similar to note title in the settings of the plugin? 
![image](https://github.com/liamcain/obsidian-periodic-notes/assets/37670665/9b7c68a3-98d8-43cf-9143-bea2b12dc699)
Or is there another way to make it so that the filename has this calculation?

**Comment by arnoesterhuizen** [comment]
Yep, those are used in the template, but I haven't thought of trying it in the filenames. Try adding it an see if it works?
